bittensor/_axon/__init__.py

```diff
@@ -227,13 +227,13 @@ def add_args( cls, parser: argparse.ArgumentParser, prefix: str = None  ):
     def add_defaults(cls, defaults):
         """ Adds parser defaults to object from enviroment variables.
         """
-        defaults.axon = bittensor.config()
+        defaults.axon = bittensor.Config()
```

Comment:
Is there a reason that the axon config needs to be the actual class rather than the factory class?

Reply:
this is what every other class does in add_defaults. Basically the config factory instantiates a new argument parser instance.
This messed up the argparser creation and makes btcli --help not work.

 What you're supposed to do is create a new Config object (Munch object) so config.axon works after add_defaults (is a Munch object)